scripts/pipelines/gcloud/templates/package/package-pipeline.yml.template

```
@@ -0,0 +1,18 @@
+steps:
+- name: 'gcr.io/cloud-builders/gsutil'
+  args: ['cp', 'gs://package-test-bucket/build_1.tar', '.']
```

The name and path is not good
```suggestion
  args: ['cp', 'gs://${PROJECT_ID}_cloudbuild/pipelinesArtifacts/${buildPipelineName}/commit_${SHORT_SHA}', '.']
```